assets/images/logo/README.md

```diff
@@ -0,0 +1,30 @@
+# Logo
+
+Authors: code3z (@code3 on the forum) and ortiza5 (@nomadic on the forum)
+
+⚠️ The content in this folder and subfolders is NOT in the public domain and the Unlicense does NOT apply.
+No code should be stored in this folder.
+
+Licensed under a
+[Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/).
+ Credits to Vivaldi Technologies, modding-vivaldi Authors, and ChromeDevTools Logo Creators.
+
+The "V" in the logo is trademarked and should not be used commercially or to imitate Vivaldi Technologies.
+
+Based on:
+https://github.com/ChromeDevTools/devtools-logo/blob/master/Logo.svg
+
+Permissions:
+https://web.archive.org/web/20210601045407if_/https://forum.vivaldi.net/topic/61636/legal-usage-of-vivaldi-logo/8
```

Comment:
Sorry, I forgot to check the middle part of the URL. It seems that the `if_` is not needed.

Reply:
Ok